openedx/features/content_type_gating/field_override.py

```diff
@@ -46,6 +46,14 @@ def get(self, block, name, default):
 
         if original_group_access is None:
             original_group_access = {}
+
+            # For Feature Based Enrollments, we want to inherit group access configurations
+            # from parent blocks. The use case is to allow granting access
+            # to all graded problems in a unit at the unit level
+            merged_group_access = block.get_parent().merged_group_access
```

Comment:
Will this cause similar "dirty" issues we saw before?

Reply:
I *think* this is ok, because `merged_group_access` is a property, not an xblock field. It does access fields, though... still, it works elsewhere in the system, so I'm not sure why it wouldn't work here.